test/unit/services/testCoverSheetPdf.js

```diff
@@ -140,7 +140,7 @@ describe('CoverSheetPdfService', () => {
                 will: {
```

Comment:
Are there any component tests that need to be changed/added?

Reply:
i've not update any component and just use the component, so no need to add any new test for the component

Reply:
ok